src/main/java/com/tvtracker/enterprise/dao/IMediaEntryDAO.java

```diff
@@ -37,7 +37,7 @@
          * @param id integer uniquely identifying a MediaEntry record
          * @return MediaEntry object representation of corresponding database record
          */
-        MediaEntry fetch(int id) throws SQLException, IOException, ClassNotFoundException;
+        MediaEntry fetchByID(int id) throws SQLException, IOException, ClassNotFoundException;
```

Comment:
Changed the name to fetchByID because it more encapsulates what it does.
Source: https://www.baeldung.com/java-clean-code
https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3

Reply:
Either one works here.  Since the implicit parameter is named "id", the nature of fetch by ID is a given.